File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>
// This program prints mario's block when given an input of 1-23
 


int main(void)
{    
    int userInput;
    do
    { 
        printf("Height\n");

        userInput = get_int();
    }
    while(userInput < 0 &&userInput < 23);
    
    int hash = 1;
    int space = userInput - 1;
    
    for(int i = 0; i < userInput; i++)
    {
        
             for(int j= 1; j<= space; j++)
             {
                 printf(" ");
             }
             for(int k= 0; k< hash; k++)
             {
                 printf("#");
             }
             
             printf("\n");
             space--;
             hash ++;

    }
}
     





     
     
     

File: pytho/mario.py
from cs50 import get_int
while True:
    print("Height")
    userInput = get_int()
    if (userInput > 0) and (userInput < 23):
        break
hash = 1
space = userInput - 1
for i in range(userInput):
    for j in range(space):
        print(" ",end="")
    for k in range(hash):
        print("#", end="")
    print()
    space = space - 1
    hash = hash + 1
